src/integrationTest/resources/db/migration/V1_9__create_tables.sql

```diff
@@ -0,0 +1,163 @@
+	CONSTRAINT region_id_pk PRIMARY KEY (region_id)
+);
+
+create table building_location(
+	building_location_id varchar(16),
+	epimms_id varchar(16),
+	building_location_name varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	building_location_status_id varchar(16),
+	area varchar(16),
+	region_id varchar(16),
+	cluster_id varchar(16),
+	court_finder_url varchar(512),
+	postcode varchar(8),
+	address varchar(512),
+	constraint building_location_pk primary key (building_location_id),
+	constraint epimms_id_uq unique (epimms_id)
+);
+
+create table district_family_jurisdiction (
+	district_family_jurisdiction_id varchar(16) NOT NULL,
+	description varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	welsh_description varchar(256),
+	CONSTRAINT district_family_jurisdiction_id_pk PRIMARY KEY (district_family_jurisdiction_id)
+);
+
+create table court_district_family_jurisdiction_assoc(
+  court_district_family_jurisdiction_assoc_id varchar(16) NOT NULL,
+  district_family_jurisdiction_id varchar(16),
+	court_location_id varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_district_family_jurisdiction_assoc_id_pk PRIMARY KEY (court_district_family_jurisdiction_assoc_id));
+
+create table district_civil_jurisdiction (
+	district_civil_jurisdiction_id varchar(16) NOT NULL,
+	description varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	welsh_description varchar(256),
+	CONSTRAINT district_civil_jurisdiction_id_pk PRIMARY KEY (district_civil_jurisdiction_id)
+
+);
+
+create table court_district_civil_jurisdiction_assoc(
+  court_district_civil_jurisdiction_assoc_id varchar(16) NOT NULL,
+  district_civil_jurisdiction_id varchar(16),
+	court_location_id varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_district_civil_jurisdiction_assoc_id_pk PRIMARY KEY (court_district_civil_jurisdiction_assoc_id));
+
+create table cluster (
+	cluster_id varchar(16) NOT NULL,
+	cluster_name varchar(256),
+	welsh_cluster_name varchar(256),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT cluster_id_pk PRIMARY KEY (cluster_id)
+);
+
+create table court_location_category(
+  court_location_category_id varchar(16) NOT NULL,
+	court_location_category varchar(128),
+	welsh_court_location_category varchar(128),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_location_category_id_pk PRIMARY KEY (court_location_category_id)) ;
+
+create table court_location_category_service_assoc(
+  court_location_category_service_assoc_id varchar(16) NOT NULL,
+  service_code varchar(16),
+	court_location_category_id varchar(16),
+	created_time timestamp,
+	updated_time timestamp,
+	CONSTRAINT court_location_category_service_assoc_id_pk PRIMARY KEY (court_location_category_service_assoc_id)) ;
+
+
+create table building_location_status(
+	building_location_status_id varchar(16),
+	status varchar(32),
+	welsh_status varchar(16),
```

Comment:
welsh_status should be 32 we have to correct in the data model. status and welsh_status should be of same size